### PR TITLE
Changes to support WIF for KubernetesManifestV1 task.

### PR DESCRIPTION
**Pull request by Deekshitha981**
**Task name**: KubernetesManifestV1 

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Review by starkmsu — APPROVED** [review]

File: Tasks/KubernetesManifestV1/_buildConfigs/Node20/package-lock.json [hunk 1]
```diff
@@ -302,22 +291,6 @@
         "uuid": "^3.0.1"
       }
     },
-    "node_modules/azure-pipelines-tasks-azure-arm-rest/node_modules/minimatch": {
```
Comment (z00sts) [inline Tasks/KubernetesManifestV1/_buildConfigs/Node20/package-lock.json:305]: So there were three entries for minimatch before?

**Review by manolerazvan — APPROVED** [review]

File: Tasks/KubernetesManifestV1/_buildConfigs/Node20/package-lock.json (hunk 1 above)
Comment (Deekshitha981) [reply Tasks/KubernetesManifestV1/_buildConfigs/Node20/package-lock.json:305]: These were auto generated.

**Review by manolerazvan — APPROVED** [review]

**Pull request merged**